__tests__/app.test.js

```diff
@@ -24,6 +24,17 @@ describe('app', () => {
     });
   });
 
+  describe('/api', () => {
+    it('should ', () => {
+      return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({ body }) => {
+          expect(typeof body).toBe('object');
```

Comment:
Fairly basic test - could make this a much better test by parsing it and checking that it's got the right properties